Swift/BRCryptoDemo/CoreDemoAppDelegate.swift

```diff
@@ -191,5 +193,28 @@ extension UIApplication {
             app.system.managers.forEach { $0.connect() }
         }
     }
+
+    static func reset () {
+        guard let app = UIApplication.shared.delegate as? CoreDemoAppDelegate else { return }
+        print ("APP: Resetting")
+
+        // Create a new system
+        let system = System (listener: app.system.listener!,
+                             account: app.system.account,
+                             path: app.system.path,
+                             query: app.system.query)
+
+        // Stop the existing system
+        app.system.stop()
+        app.summaryController.reset()
+
+        
+        // Assign and then configure the new system
+        app.system = system
+        app.system.configure()
+
+        // Start
+        system.start(networksNeeded: [])
```

Comment:
I thought start was removed, does this build?

Reply:
Drat.

Reply:
Fixed; ran and Crypto unit tests pass (Core test fail on a ETH P2P issue).